src/esp/physics/ArticulatedObject.h

```diff
@@ -497,6 +497,15 @@ class ArticulatedObject : public esp::physics::PhysicsObjectBase {
    */
   virtual std::vector<float> getJointPositions() { return {}; }
 
+  /**
+   * @brief Get the torques on each joint
+   *
+   * @return Array of torques on each joint
```

Comment:
Technically returns a vector of torques. You can return an array and it will be converted by PyBind11 as a numpy array